src/js/view/controls/templates/dock.js

```diff
@@ -16,8 +16,8 @@ const dockButton = (buttonClass, buttonId, image, tooltipText) => {
     const style = image ? `style='background-image: url("${image}")` : '';
     const tooltipHtml = tooltipText ? tooltip(tooltipText) : '';
     return (
-        `<div class="jw-dock-button jw-background-color jw-reset ${buttonClass || ''}" button="${buttonId}">` +
-            `<div class="jw-icon jw-dock-image jw-reset" ${style}</div>` +
+        `<div class="jw-dock-button jw-background-color jw-reset ${buttonClass || ''}" button=${buttonId}">` +
+            `<div class="jw-icon jw-dock-image jw-reset ${style}"></div>` +
```

Comment:
This is incorrect. `style=` is a new attribute - not part of the class attribute.